Traceback (most recent call last):
  File "/scratch/jmille16/tylers_code/gpar2.py", line 130, in <module>
IOError: [Errno 4] Interrupted system call: '/scratch/jmille16/tylers_code/temp/saveplace_86.txt'
